src/stageros.cpp

```diff
@@ -354,51 +352,56 @@ StageNode::SubscribeModels()
                 new_robot->cameramodels.push_back(this->cameramodels[s]);
                 this->cameramodels[s]->Subscribe();
 
-		ROS_INFO( "subscribed to Stage camera model \"%s\"", this->cameramodels[s]->Token() ); 
+        RCLCPP_INFO(n_->get_logger(),  "subscribed to Stage camera model \"%s\"", this->cameramodels[s]->Token() );
```

Comment:
The 3 logs above have some weird indentation (although that was already the case).

Reply:
Fixed.